src/StorageController.weapp.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2015-present, Parse, LLC.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+localStorage = {
```

Comment:
Is there a reason why you are overriding localStorage? Can you use wx.getStorageSync and others directly below?

Reply:
The Wechat miniprogram does not support localStorage，https://developers.weixin.qq.com/miniprogram/en/dev/api/storage/wx.setStorageSync.html

Reply:
You can remove this and just use wx directly no? It’s basically a wrapper.